text/0020-contributing-guidelines.md

```diff
@@ -0,0 +1,46 @@
+# Contributing guidelines
+- Start Date: 2019/11/15
+- RFC PR: https://github.com/openmrs/openmrs-rfc-frontend/pull/20
+
+## Decision, including impact on distributions
+For frontend projects within OpenMRS, the following are used to instruct contributors on how to make a pull request.
+
+- [Github contributing.md files](https://help.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors#adding-a-contributing-file)
+- [Github pull_request_template.md files](https://help.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository)
+- [The OpenMRS code of conduct](https://wiki.openmrs.org/display/docs/Code+of+Conduct)
+- [The OpenMRS MPL-2.0 + Health care disclaimer software license](https://wiki.openmrs.org/display/RES/OpenMRS+License+FAQ)
+
+The [OpenMRS Pull Request Tips](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) are recommended.
+
+[Github squash and merge](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges#squash-and-merge-your-pull-request-commits) is enabled for all frontend projects.
+
+For frontend projects within OpenMRS, the following guidelines apply:
+- Not all PRs have to link to a JIRA ticket.
+- Pull requests are allowed (and encouraged) to contain more than one commit. Those commits are squashed when merged into the master branch.
```

Comment:
Totally agree that with the squash and merge functionality that Github provides the one-commit-per-PR rule is outdated.